File: UCF HSPT Project/src/y2015/maximum.java
import java.util.*;

public class maximum 
{
   public static void main(String[] args) 
   {
      Scanner in = new Scanner(System.in);

      int max = 0;
      int N = in.nextInt();
      for (int i=0; i<N; i++) {

         int x = in.nextInt();

         // If this is the highest number so far, output it and mark
         // it as our new max
         if (x > max)
         {
            System.out.printf("%d!%n", x);
            max = x;
         }
      }
   }
}

File: UCF HSPT Documents/2015/Solutions/maximum.cpp

#include <stdio.h>

int main()
{
   // Read the number of integers
   int tests;
   scanf("%d", &tests);

   // Loop through each integer
   int max;
   max = 0;
   for (int t = 1; t <= tests; t++)
   {
      // Read in the integer
      int x;
      scanf("%d", &x);

      // If this is the largest integer so far, output it and
      // mark it as the new max
      if (x > max)
      {
         printf("%d!\n", x);
         max = x;
      }
   }

   return 0;
}
